tools/scripts/prplMesh-platform-init.sh

```diff
@@ -0,0 +1,79 @@
+
+#!/bin/bash
+#
+
+echr() {
+	echo -e '\033[1;31m'"$@"'\033[0m'
+}
+
+if [ `id -u` -ne 0 ]; then
+    echr "$0: This script must be run as root"
+    exit 1
+fi
+
+VERBOSE=false
+HELP=false
+HOSTAPD=`which hostapd`
```

Comment:
Redirect error to /dev/null. I don't have hostapd in my $PATH, so this prints an error even if I pass the -H option.
```suggestion
HOSTAPD=`which hostapd 2>/dev/null`
```

Reply:
Done